### DISCUSSION TITLE
Question: How do you add custom stages?

### DISCUSSION THREAD

**Discussion by amogus13** [FunkinCrew/Funkin#1528] [Q&A]
I tried copying code from pico's stage but it doesn't work.
```
case 'sussy': 
{
curStage = 'amogus';
var bg:FlxSprite = new FlxSprite(-10).loadGraphic(Paths.image('back'));
bg.scrollFactor.set(0.1, 0.1);
add(bg);          
}
```
![image](https://user-images.githubusercontent.com/64611348/115177975-23675080-a085-11eb-87a4-07493c1567d9.png)

**Comment by R-code-0** [comment]
Edit storyMenu

**Comment by amogus13** [comment]
> Edit storyMenu

Not talking about adding weeks

**Comment by R-code-0** [comment]
> > Edit storyMenu
> 
> Not talking about adding weeks

What problem? Are you sure your code is right? Try to copy from other stage. Pico has much sprites.

**Comment by amogus13** [comment]
> > > Edit storyMenu
> > 
> > 
> > Not talking about adding weeks
> 
> What problem? Are you sure your code is right? Try to copy from other stage. Pico has much sprites.

The problem is that the background wont show up and the other week's code are pretty much all the same but when I use it on my week it does nothing. 
Updated code: 
```
case 'sussy': 
{
curStage = 'amogus';
var backsus:FlxSprite = new FlxSprite().loadGraphic(Paths.image('back'));
backsus.scrollFactor.set(0.1, 0.1);
backsus.updateHitbox();
add(backsus);

var frontsus:FlxSprite = new FlxSprite().loadGraphic(Paths.image('front'));
frontsus.scrollFactor.set(0.1, 0.1);
frontsus.updateHitbox();
add(frontsus);		  
}
```

**Comment by R-code-0** [comment]
Where is located your bg and what week is it?

**Comment by SilverT4** [comment]
Try setting your scroll factor to **0.9** instead of 0.1

**Comment by amogus13** [comment]
> Where is located your bg and what week is it?

Its a custom week I added its located in its own week7 folder

**Comment by amogus13** [comment]
> Try setting your scroll factor to **0.9** instead of 0.1

didn't work
